src/poprox_storage/repositories/demographics.py

```diff
@@ -56,6 +57,31 @@ def fetch_demographics_by_account_ids(self, account_ids: list[UUID]) -> list[Dem
             )
             for row in result
         ]
+        
+    # fetching latest demographic info
+    def fetch_latest_demographics_by_account_id(self, account_id: UUID) -> Demographics:
+        demographics_tbl = self.tables["demographics"]
+
+        demo_query = (
+            select(demographics_tbl)
+            .where(demographics_tbl.c.account_id == account_id)
+            .order_by(demographics_tbl.c.created_at.desc())
+            .limit(1)
+        )
+        result = self.conn.execute(demo_query).fetchone()
+
+        if result is None:
+            return None
+
+        return Demographics(
+            account_id=result.account_id,
+            gender=result.gender,
+            birth_year=result.birth_year,
+            zip5=result.zip3,
```

Comment:
this line looks funky.

Reply:
Same issue here: the code is right but the attribute name is wrong